controllers/lvmcluster_controller_test.go

```diff
@@ -100,6 +110,13 @@ var _ = Describe("LVMCluster controller", func() {
 				return errors.IsNotFound(err)
 			}, timeout, interval).Should(BeTrue())
 
+			// auto deletion of VgManager resource based on CR deletion
+			By("confirming absence of VgManager Resource")
```

Comment:
This will not work. See https://book.kubebuilder.io/reference/envtest.html#testing-considerations)

Reply:
Since no garbage collection in envTest, just checking if the VgManager has the right owner reference.